### ISSUE TITLE
aria-labelledby + title on <select> element

### ISSUE THREAD

**Issue by devarshipant** [FreedomScientific/VFO-standards-support#79]
Summary
> aria-labelledby + title on `<select>` element -- jaws 18 / 2018 lack support in IE 11

Example:
> 1. https://codepen.io/devpant/pen/KRqwoG

### Expected result

> On select element, title attribute should be read after the aria-labelledby idref is read in IE11.

### Actual result

> title should be read after aria-labelledby idref. 

**Note: it reads properly in Chrome.**

### Example

https://codepen.io/devpant/pen/KRqwoG

**Comment by JAWS-test** [comment]
The problem no longer exists with JAWS 2019.
